internal/workflow/processing.go

```diff
@@ -1188,3 +1208,72 @@ func (w *ProcessingWorkflow) sendToFailedBucket(
 
 	return nil
 }
+
+func (w *ProcessingWorkflow) validatePREMIS(
+	ctx temporalsdk_workflow.Context,
+	xmlPath string,
+	paID int,
+) error {
+	id, err := w.createPreservationTask(
```

Comment:
**Now that I see the tests, we forgot to check the enabled config before running all this. It should be easy, just check the config at the start and return nil directly if it is not enabled.**

After adding that, all the tests won't run this activity unless you enabled the validation in each test configuration. You can only enable it in one of the tests, to avoid making changes to all of them.